[2017-06-06 16:03:05 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:03:05 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmpcEG2As If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:03:05 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:03:05 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:06:51 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:07:09 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:07:39 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:07:39 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmpoa9CyL If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:07:40 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:07:40 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:07:40 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:09:02 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:09:33 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:09:33 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmprQfmLf If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:09:33 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:09:33 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:09:34 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:11:10 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:11:40 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:11:40 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmpIFgfx8 If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:11:41 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:11:41 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:13:02 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:15:10 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:15:40 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:15:40 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmp7nW7_T If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:15:40 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:15:40 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:17:00 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:17:03 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:17:34 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:17:34 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmpbydZ9W If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:17:34 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:17:34 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:17:40 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:18:10 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:18:10 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmph4KhPz If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:18:10 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:18:10 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:18:13 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:44:02 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:44:32 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:44:32 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 188, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmp01ixyD If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:44:33 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:44:33 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 232, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 16:45:55 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 16:46:12 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 16:46:42 driver.web.selenium] ERROR Open browser failed.
[2017-06-06 16:46:42 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 189, in __get_page
    self.__root = webdriver.Firefox(executable_path=driver_path)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
    self.binary, timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
    self.binary.launch_browser(self.profile, timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
    self._wait_until_connectable(timeout=timeout)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
    % (self.profile.path))
WebDriverException: Message: Can't load the profile. Profile Dir: /var/folders/t4/tx3gyc293dndkm82gpx200540000gn/T/tmpoD96sZ If you specified a log_file in the FirefoxBinary constructor, check it for details.


[2017-06-06 16:46:42 driver.web.selenium] ERROR 'NoneType' object has no attribute 'save_screenshot'
[2017-06-06 16:46:42 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 81, in start
    _result = self.__execute(_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 115, in __execute
    self.save_screen()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcDriver/Web/WebDriver.py", line 233, in save_screen
    self.__root.save_screenshot(self.__pic_name)
AttributeError: 'NoneType' object has no attribute 'save_screenshot'

[2017-06-06 17:10:26 driver.web.selenium] INFO Run command {u'quit': u'QUIT'}
[2017-06-06 17:10:32 driver.web.selenium] INFO Run command {u'TYPE': u'PAGE', u'OBJECT': u'300000001', u'ENV': u'TEST', u'OPERATION': u'GET', u'DATA': None, u'BROWSER': u'FIREFOX'}
[2017-06-06 17:11:02 driver.web.selenium] ERROR Open browser failed.
